hwsc_file_transaction_svc.py

```diff
@@ -9,67 +9,79 @@
 
 class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
     def __init__(self):
 
         class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
             def __init__(self):
-                # pass
-                self.tmp_file_name = ''
+                pass
+
 
-            def GetStatus(self, request, context):
+            def get_status(self, request, context):
                 print("Get Status")
 
-            def download(self,downloadRequest,context):
-                if downloadRequest.name:
+            def download_zip_files(self, request_iterator, context):
+                if request_iterator.name:
                     return download_chunk(self.tmp_file_name)
 
-            def upload(self,uploadRequest,context):
-                upload_chunk(uploadRequest,self.tmp_file_name)
-                return hwsc_file_transaction_svc_pb2.Reply(length=os.path.getsize(self.tmp_file_name))
+          #  def UploadFile(self, request_iterator, context):
+           #     print("[INFO] Requesting UploadFile service")
+                # save_chunks_to_file(request_iterator, context.fileName)
+                # print(request_iterator.Chunk.fileName)
+            #    print(context.Chunk.fileName)
+             #   return hwsc_file_transaction_svc_pb2.FileTransactionResponse()
+                # if not request_iterator.buffer.is_valid or request_iterator.fileName.is_valid:
+                #     message = 'Upload Error!'
+                #     context.set_details(message)
+                #     context.set_code(grpc.StatusCode.INVALID_ARGUMENT)
+                #     return hwsc_file_transaction_svc_pb2.FileTransactionResponse()
+                #
+                # status = hwsc_file_transaction_svc_pb2.FileTransactionResponse()
+                # status.code = grpc.StatusCode.OK
+                # assert status.HasField("code")
+                #
+                # return hwsc_file_transaction_svc_pb2.FileTransactionResponse(
+                #     message='OK',
+                #     status=status.code,
+                #     url='url: /res' + context.fileName,
+                #     length=64)
+
+            def upload_file(self, request_iterator, context):
+                print("[INFO] Requesting UploadFile service")
+
+                for getName in request_iterator:
+                    save_chunks_to_file(request_iterator, getName.fileName)
+
+                return hwsc_file_transaction_svc_pb2.FileTransactionResponse(message='OK')
+
 
             self.server = grpc.server(futures.ThreadPoolExecutor(max_workers=1))
 
         hwsc_file_transaction_svc_pb2_grpc.add_FileTransactionServiceServicer_to_server(Servicer(), self.server)
 
     def start(self, port):
-        print("1")
-
         self.server.add_insecure_port(f'[::]:{port}')
-        print("2")
-
         self.server.start()
-        print("3")
+        print("Server is running...")
```

Comment:
Make the start sequence printouts similar to https://github.com/faraonc/hwsc-document-svc/blob/master/main.go